src/content/post/launching-trieve-hn-discovery.mdx

```diff
@@ -0,0 +1,101 @@
+---
+publishDate: 2024-08-29T08:45:00.000Z
+author: skeptrune (Nick K)
+title: >-
+  Launching Trieve HN Discovery - A Discovery Focused Search Engine for Hacker
+  News
+excerpt: >-
+  Trieve HN Discovery is an open source discovery-focused search engine for
+  Hacker News. With features like LLM-generated query suggestions, site filters,
+  semantic search, RAG AI chat, and order by descendant count, it offers a
+  comprehensive search experience for the Hacker News community.
+image: >-
+  https://cdn.trieve.ai/blog/trieve-hn-discovery/trieve-hn-discovery-preview-opengraph.webp
+category: Demos
+tags:
+  - hackernews
+  - search
+  - demo
+---
+
+We have been interested in building a HN discovery interface since before we were accepted into the W24 Y-Combinator batch in late 2023\. However, we were limited by our own product not being good enough to develop what we wanted. Our ingest speed and reliability in particular were extreme challenges. Now, [after 6 months of hacking in our spare time](https://github.com/devflowinc/trieve-hn-discovery/commit/2e24998d30b336416a947ee1594eb9c026958087), we are excited to share the complete version available at [hn.trieve.ai](https://hn.trieve.ai)!
```

Comment:
?: Y-Combinator => Y Combinator

Reply:
?: => Now, with a production-ready Trieve backend and [6 months of hacking on this in our spare time](https://github.com/devflowinc/trieve-hn-discovery/commit/2e24998d30b336416a947ee1594eb9c026958087)